apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.Settings.cs

```diff
@@ -65,6 +65,20 @@ public sealed class Settings
         /// </summary>
         public TimeSpan? AckDeadline { get; set; }
 
+        /// <summary>
```

Comment:
It feels a little odd for a minimum and maximum to be configured by the user. I'd normally expect the user to specify an AckDeadlineExtension, and for that to be constrained by a system min/max value. This may all make sense when I come to the code later, but it's work thinking about in terms of user understanding. How does this relate to `AckDeadline` above? Is there a difference between `AckDeadline` and `AckDeadlineExtension`?